omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -116,6 +116,54 @@
 default['private_chef']['rabbitmq']['consumer_id'] = 'hotsauce'
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
+default['private_chef']['rabbitmq']['ssl_versions'] = "'tlsv1.2', 'tlsv1.1'"
+
+####
+# RabbitMQ Management Plugin
+####
+default['private_chef']['rabbitmq']['management_user'] = "rabbitmgmt"
+default['private_chef']['rabbitmq']['management_password'] = "chefrocks"
+default['private_chef']['rabbitmq']['management_port'] = 15672
+default['private_chef']['rabbitmq']['management_enabled'] = true
+
+# RabbitMQ max-length policy
+default['private_chef']['rabbitmq']['analytics_max_length'] = 10000
+default['private_chef']['rabbitmq']['queue_length_monitor_vhost'] = "/analytics"
+default['private_chef']['rabbitmq']['queue_length_monitor_queue'] = "alaska"
+default['private_chef']['rabbitmq']['queue_length_monitor_enabled'] = true
+
+####
+# RabbitMQ Queue Monitor
+####
+# how often to run the queue monitor
+default['private_chef']['rabbitmq']['queue_length_monitor_millis'] = 30000
+# if the queue monitor is busy and this timeout has been exceeded,
+# assume that rabbit is in a bad state and don't send messages to it
+# 5000 is the default of gen_server:call()
+default['private_chef']['rabbitmq']['queue_length_monitor_timeout_millis'] = 5000
+
+# don't send messages to rabbitmq if it has reached it's configured max_length
+default['private_chef']['rabbitmq']['drop_on_full_capacity'] = true
+
+# prevent erchef from starting if queue is at capacity
+default['private_chef']['rabbitmq']['prevent_startup_on_full_capacity'] = true
```

Comment:
This is concerning to me for two reasons:
1)  it is in the rabbitmq namespace, but it actually impacts erchef without the word `erchef` anywhere in the variable
2)  Is this the default behavior we want?   I think this comes down to a discussion on "fail open vs fail closed" -  and I believe our users would prefer we default to "fail open"  (prefer API availability)